crates/mun_runtime_capi/src/lib.rs

```diff
@@ -55,14 +56,7 @@ pub unsafe extern "C" fn mun_runtime_create(
         )));
     }
 
-    let library_path = match CStr::from_ptr(library_path).to_str() {
```

Comment:
These are C-style strings provided by the user, whereas the C-style strings in the ABI are generated by our own compiler. As we cannot provide any guarantees about the validity of the former, we still want to do runtime checks.

Reply:
I revert this change. Thanks for explanation!